harvester_e2e_tests/fixtures/network.py

```diff
@@ -127,9 +127,13 @@ def network(request, admin_session, harvester_api_endpoints, enable_vlan):
     yield network_data
 
     if not request.config.getoption('--do-not-cleanup'):
-        _cleanup_network(admin_session, harvester_api_endpoints,
-                         network_data['id'],
-                         request.config.getoption('--wait-timeout'))
+        # XXX: we would need to check the network not be deleted terraform yet
+        if not utils.is_marker_enabled(request, 'terraform') and \
+            _lookup_network(request, admin_session, harvester_api_endpoints,
+                            vlan_id) is not None:
+            _cleanup_network(admin_session, harvester_api_endpoints,
```

Comment:
Ahh, this is smart, to check to make sure the Network exists before we attempt to clean it up!
If so, should we indent this method call to be within the nested result of the `_lookup_network() is not None:` conditional?
Something like:
```
        if not utils.is_marker_enabled(request, 'terraform') and \
            _lookup_network(request, admin_session, harvester_api_endpoints,
                            vlan_id) is not None:
                _cleanup_network(admin_session, harvester_api_endpoints,
                                 network_data['id'],
                                 request.config.getoption('--wait-timeout'))
```
But maybe it's just a flake8 formatting thing, I'm not sure.

Reply:
No, we don't need to make another indent for the `_cleanup_network...`.
when you look into Line 131-133, it means:
```python
if not utils.is_marker_enabled(request, 'terraform') and _lookup_network(request, admin_session, harvester_api_endpoints, vlan_id) is not None:
    _cleanup_network(....)
```
In the end of Line 131, we use backslash(`\`) as the line continuation character, but we don't need to another backslach(`\`) for Line 132, because it is wrapped in parentheses.

Reply:
Ahh you're right I completely missed the `\` 😅